joint_limits/include/joint_limits/joint_limiter_struct.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 PAL Robotics S.L.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// \author Sai Kishor Kothakota
+
+#ifndef JOINT_LIMITS__JOINT_LIMITER_STRUCT_HPP_
+#define JOINT_LIMITS__JOINT_LIMITER_STRUCT_HPP_
```

Comment:
I'd suggest a slightly different filename, something that isn't so explicit yet vague

`include/joint_limits/data_structures.hpp`? still vague but at least it only gives you a hint, not the expectation of finding `struct JointLimiter` in here

Reply:
Sure. Very reasonable

Reply:
Done